developers/academy/js/starter_multimodal_data/102_mm_collections/20_create_collection.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Create a collection
+description: Creating Multimodal Data Collections
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import TSCode from '!!raw-loader!../_snippets/102_collection.mts';
+
+Weaviate stores data in "collections". A collection is a set of objects that share the same data structure. In our movie database, we might have a collection of movies, a collection of actors, and a collection of reviews.
+
+Here we will create a collection of movies.
+
+## <i class="fa-solid fa-code"></i> Code
+
+This example creates a collection for the movie data:
+
+<FilteredTextBlock
+  text={TSCode}
+  startMarker="// CreateMovieCollection"
+  endMarker="// END CreateMovieCollection"
+  language="ts"
+/>
+
+Each collection definition must have a name. Then, you can define additional parameters like we've done in this example.
+
+## <i class="fa-solid fa-chalkboard"></i> Explain the code
+
+### <i class="fa-solid fa-chalkboard"></i> Properties
+
+Properties are the object attributes that you want to store in the collection. Each property has a name and a data type.
+
+In our movie database, we have properties like `title`, `release_date` and `genre_ids`, with data types like `TEXT` (string), `DATE` (date), or `INT` (integer). It's also possible to have arrays of integers, like we have with `genre_ids`.
+
+As a multimodal object, we also have the `poster` property which is the image data, which is saved as a `BLOB` (binary large object) data type.
+
+#### Auto-schema
+
+Weaviate can automatically [infer the schema](/developers/weaviate/config-refs/schema/index.md#auto-schema) from the data. However, it's a good practice to define the properties explicitly, for better control and to avoid surprises.
+
+### <i class="fa-solid fa-chalkboard"></i> Vectorizer configuration
+
+If you do not specify the vector yourself, Weaviate will use a specified vectorizer to generate vector embeddings from your data.
+
+In this code example, we specify the `multi2vec-cohere` module. This module uses the Cohere Multimodal Embed 3 model to generate vector embeddings from the text and image data.
```

Comment:
This is true now - but may not be in the future. 

I would highlight that this is the current default model. Tbh I think it would be best to explicitly specify the model in the vectorizer. 

(In the equivalent Python one, the model is specified in the Docker file due to the nature of the CLIP integration.)

Reply:
Ah makes sense, will do!